### ISSUE TITLE
Incorrect usage of role in Menu

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/uikit#1342]
Element with role "menu" cannot have "li" tags.

```
Element has children which are not allowed: li[tabindex]
```

https://preview.gravity-ui.com/uikit/?path=/story/components-navigation-menu--default
